### PR TITLE
Packaging updates

### PR DESCRIPTION
**Pull request by pluflou**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+zip_safe = false
+include_package_data = true
+
+[tool.setuptools.packages.find]
+where = ["src"]
+include = [ "badger", ]
+namespaces = false
+
+[tool.ruff]
+# select = []
```
Comment (YektaY) [inline pyproject.toml:71]: Are the commented out select and ignore lists being left in for a future update?

**Review by YektaY — APPROVED** [review]
This looks great, I only have one small question on some commented out lines in the toml file.

File: pyproject.toml (hunk 1 above)
Comment (pluflou) [reply pyproject.toml:71]: Yes I am working on adding linting using ruff now. So that will come later this week.

**Comment by pluflou** [comment]
@YektaY I just removed the MANIFEST.in since `setuptools_scm` will tell `setuptools` to add all files tracked by git to the sdist, which replaces what we have in the file

**Pull request merged**